js/add-books.js

```diff
@@ -70,6 +70,11 @@ const display = () => {
     }
     contain.appendChild(bookContainer);
   });
+
+  if (books.length !== 0) {
+    document.querySelector('.all-awesome').style.display = 'block';
+    document.querySelector('.list-book').style.border = '2px solid #000';
+  }
 };
 
 /* eslint-disable no-unused-vars */
```

Comment:
- [ ] please avoid turning off linters they are used to make your project work better and easier to read.

Reply:
Hello @mikethreels,

I told you in the requirements they give us the possibility to turn off this kind of linter error 
Why do you ask me that again?

<img width="1254" alt="Screen Shot 2023-07-04 at 5 12 28 PM" src="https://github.com/jecode93/awesome-book/assets/45808742/6356d694-e5da-42a1-b12f-f8c7a3bcd13e">

So you do that to keep us from moving forward or what?

Reply:
ooh @mikethreels 
I send you the picture it's still in the same project, not another one, why are you so strict with that as I send you the picture?

I think you can read it here [projects requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects_v1_1.md), it's the whole project, not just one things...

Reply:
I think you do that to keep us from moving forward.

Reply:
Dear @jecode93 
Please rest assured that the only reason I'm requesting any changes is because I'm trying to help you make your project as good as possible. I would like to ask you not to accuse me of blocking your project while I'm spending a lot of time trying to help you improve.

Thank you for sharing the screenshot, however, looking at the title and URL in your print screen I can see you're sharing the requirements of the "Plain JS object" milestone even though you requested a review for the "full website" milestone.

Is it possible you requested a review for a different milestone than the one you intended?

Happy coding!
Michael Threels

Reply:
@mikethreels do you know that it is not a question about a milestone but it's about the whole project!?

It's just an error that is not relevant to the project requirements as well!

I understand that you spent a lot of time trying to help me, but I think I spent a lot of time too trying to work on this project I could understand if the project didn't gives us the possibility but they give us the possibility to turn it off so I think this cannot keep the project from approving!